### ISSUE TITLE
can not boot from sd card

### ISSUE THREAD

**Issue by hitqiu** [balister/oe-gnuradio-manifest#3]
thank you for great job you have done. I have got a compressed root filesystem tarball as well as kernel and bootloader binaries available in my work/deploy directory。 But when I copy the file to the sd card(parted FAT32(boot) EXT4(rootfs)),the redboard can not boot from sd card。
if I use the boot.bin file which download from Xilinux, The board works well. what's wrong?

**Comment by balister** [comment]
Do you mean zedboard? What MACHINE are you building for?

**Comment by hitqiu** [comment]
Yeah, MACHINE ="zedboard-zynq7",The uboot.bin doesn't work. I downloard the uboot.bin file from ,which can boot the image I make.

**Comment by hitqiu** [comment]
from http://www.wiki.xilinx.com/Zynq+Releases ,the Zynq 2015.2 Release (latest) one

**Comment by hacene182** [comment]
Hi 
Me too I have the same problem 
I'am using the jethro branch.
thank you

**Comment by balister** [comment]
/me grumbles. Building from my jethro manifest now. I'll try boot testing them :) These should just work from meta-xilinx :(

**Comment by hacene182** [comment]
Hi, Philip 
I did build successfully the linux kernel for the zedboard using the jethro manifest, and I got all the necessary files (boot.bin-zedboard-zynq7, u-boot ...etc) but when I put them in the sd card to boot them on the zedboard as showed in this link (http://gnuradio.org/redmine/projects/gnuradio/wiki/Zynq),
![untitled](https://cloud.githubusercontent.com/assets/11305122/14784264/f01e9362-0af4-11e6-8553-9e87dc7b6e8e.png)
still doesn't boot !!!

If you also did build from your jethro manifest or any resent version of embedded linux with gnuradio on it , can you share with us the needed SD card files (boot.bin devicetree...etc) for the zedboard. 

Thank you.

**Comment by balister** [comment]
OK, I just did a Jethro build and it booted. I've switched the default branch shown on github to Jethro.I'm going to update the image files in my dropbox and double check them also. Sorry for the trouble.

**Comment by hacene182** [comment]
Thank you very much Sir for your time.

**Comment by hacene182** [comment]
Where can I find the Dropbox link when you finish uploading the files ?

**Comment by balister** [comment]
The new files are here.
https://www.dropbox.com/sh/yfbpj63pcenqatr/AAAt0s3xFXs47I7q5pNopheHa?dl=0
I'm not going to be able to test them for a bit, let me know if any boot.

Thanks

**Comment by hacene182** [comment]
Yes, it works  ,, thank you.
